roles/vdm/tasks/deploy.yaml

```diff
@@ -55,22 +55,18 @@
       when:
         - deployment_tooling == "docker"
     - name: deploy - Deploy SAS Viya
-      community.docker.docker_container:
```

Comment:
Are there still uses of community.docker with the switch to builtin.shell? Wondering about removing the references from Dependencies.md and requirements.yaml if not.

Reply:
Yes, there are 2 instances of `community.docker` in `viya4-deployment/public/roles/vdm/tasks/orchestration.yaml`. It's still being used to manage the docker login process as well as initially pulling the orchestration image.